core/domain/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id("java-library")
+    alias(libs.plugins.jetbrains.kotlin.jvm)
```

Comment:
`core:domain` module is created as a pure library module without android dependency to make it reusable in addition to android.